test/org/sagebionetworks/bridge/models/PagedResourceListTest.java

```diff
@@ -52,26 +66,34 @@ public void canSerialize() throws Exception {
                 new TypeReference<PagedResourceList<AccountSummary>>() {});
 
         assertEquals(page.getTotal(), serPage.getTotal());
-        assertEquals(page.getOffsetBy(), serPage.getOffsetBy());
-        assertEquals(page.getPageSize(), serPage.getPageSize());
-        assertEquals(page.getFilters().get("emailFilter"), serPage.getFilters().get("emailFilter"));
+        assertEquals(100, page.getPageSize());
+        assertEquals((Integer)123, page.getOffsetBy());
+        assertEquals("filterString", page.getEmailFilter());
+        
+        Map<String,Object> params = page.getRequestParams();
+        Map<String,Object> serParams = serPage.getRequestParams();
+        assertEquals(params.get("offsetBy"), serParams.get("offsetBy"));
+        assertEquals(params.get("pageSize"), serParams.get("pageSize"));
+        assertEquals(params.get("emailFilter"), serParams.get("emailFilter"));
         
         assertEquals(page.getItems(), serPage.getItems());
     }
     
     @Test
     public void offsetByCanBeNull() throws Exception {
         List<AccountSummary> accounts = Lists.newArrayListWithCapacity(2);
-        PagedResourceList<AccountSummary> page = new PagedResourceList<AccountSummary>(accounts, null, 100, 123)
-                .withFilter("emailFilter", "filterString");
+        PagedResourceList<AccountSummary> page = new PagedResourceList<AccountSummary>(accounts, null)
+                .withRequestParam("offsetBy", 123)
+                .withRequestParam("pageSize", 100)
+                .withRequestParam("emailFilter", "filterString");
```

Comment:
Hmm. After redesign, total should never be null. Maybe I'll remove this test, but I am also going to change the value to an int primitive to force setting a value.